cuappdev/eatery-blue-ios
# Analytics/Report Fixes

## Overview



Updates after merging Sergio's analytics/reports PR from last year

## Changes Made



- Analytics: re-download GoogleService-Info.plist
- Reports: update endpoint, report model, error handling

## Test Coverage



- Verify analytics with Matt
- Send reports then verify through GET request in Postman

## Related PRs or Issues



Sergio's PR: https://github.com/cuappdev/eatery-blue-ios/pull/6

## Screenshots



https://user-images.githubusercontent.com/57200368/233798918-4ee0aae7-cb3f-4ecc-8f7a-29abc10bf029.mp4
